src/components/App.jsx

```diff
@@ -120,12 +125,13 @@ class App extends Component {
         <Header />
         <article className="main">
           <Nav
-            renderStandards={this.renderStandards}
-            renderForm={this.renderForm}
+            allDiscussions={allDiscussions}
```

Comment:
I like how you did this. It makes sense that we need to have discussions and all discussions so that when filtering you can get discussions back on delete of a letter.